dev-python/pgspecial/pgspecial-1.11.10.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
```

Comment:
Does 3.9 work as well?

Reply:
Good point, I'll do some testing.

Reply:
I did some testing locally and everything looks good to me, I've added the support for `3.9` as well, although upstream doesn't explicitly mention `python3_{8,9}` as supported.

Reply:
Thanks! Often times it works without many changes needed even if the upstream does not explicitly declare support for e.g. 3.9. It's better to add 3.9 support now if possible rather than revisiting it again after a couple months :)